.github/workflows/mac-cpu-arm64-build.yml

```diff
@@ -16,22 +15,25 @@ jobs:
         with:
           submodules: true
 
-      - name: Install ninja
+      - name: Install ninja and jq
         run: |
-          brew install ninja
+          brew install ninja jq
```

Comment:
The machine pool already has jq.

It doesn't have ninja, but you can use xcode or make.

Remove this step to reduce dependencies.